capstone/src/main/webapp/script.js

```diff
@@ -196,3 +197,6 @@ function loadReponseChart() {
       document.getElementById('response-bar-chart'));
   chart.draw(stats, null);
 }
+
+google.charts.load('current', {packages: ['corechart']});
```

Comment:
Placing this code here prevents the charts from loading

Reply:
Charts API Docs recommend placing the code in script tags within the HTML file. Is there any way to configure the linter to ignore the HTML files?

Reply:
The HTML files are not the problem. The problem is that I can't mark the loadCharts function as exported so it thinks its unused.